src/System.Windows.Forms/src/System/Windows/Forms/ContainerControl.cs

```diff
@@ -44,6 +44,10 @@ public class ContainerControl : ScrollableControl, IContainerControl
 
         private AutoScaleMode _autoScaleMode = AutoScaleMode.Inherit;
 
+        // Top-level window is scaled by windows suggested rectangle during WM_DPICHANGED event.
+        // We use this flag to indicate its top-level window and is already scaled.
```

Comment:
I'm struggling to correctly interpret the comment - is the top window _scaled by Windows to the suggested rectangle_, or we scale it to _the rectangle suggested by Windows_?
If it is the former, perhaps word this as:
```suggestion
        // Top-level window is scaled by Windows during WM_DPICHANGED event.
        // We use this flag to detect a top-level window, and that it is already been scaled.
```

Reply:
Its the later. reworded.